mk/config.mk

```diff
@@ -337,6 +339,21 @@ CFG_DTB_MAX_SIZE ?= 0x10000
 # A subsequent boot stage must then merge the generated overlay DTB into a main
 # DTB using the standard fdt_overlay_apply() method.
 CFG_EXTERNAL_DTB_OVERLAY ?= n
+#
+# option b)
+# This define enables support for an OP-TEE provided DTB overlay as well as
```

Comment:
Minor: not clear that 'this define' refers to `CFG_OVERLAY_ADDR`. Perhaps the full names should be mentioned in the comments, next to "option a (or b)".